test/integration/serviceaccount/external_jwt_signer_test.go

```diff
@@ -187,11 +187,11 @@ func TestExternalJWTSigningAndAuth(t *testing.T) {
 			},
 			preValidationSignerUpdate: func(t *testing.T) {
 				t.Helper()
-				cpy := make(map[string]v1alpha1testing.KeyT)
+				cpy := make(map[string]v1testing.KeyT)
 				for key, value := range mockSigner.GetSupportedKeys() {
 					cpy[key] = value
 				}
-				cpy["kid-1"] = v1alpha1testing.KeyT{Key: pubKey1Bytes}
+				cpy["kid-1"] = v1testing.KeyT{Key: pubKey1Bytes}
 				mockSigner.SetSupportedKeys(cpy)
 				waitForDataTimestamp(t, client, time.Now())
 			},
```

Comment:
below, in addition to doing a TokenReview to make sure the token is valid, construct a client with that token and make an actual API call (I'd suggest SelfSubjectReview to fetch the self-identified user)

this will extend the integration test to fully exercise the token end-to-end, as described in the test plan for the KEP